src/service.ts

```diff
@@ -1,5 +1,8 @@
 import express, { Response } from "express";
 import httpContext from "express-http-context";
+
```

Comment:
You don't need this whitespace

Reply:
Done; while I was there, I reordered imports, so that all libraries are before local imports.